app/controllers/projects_controller.rb

```diff
@@ -68,10 +74,22 @@ def project
     @project ||= policy_scope(Project).find(params[:id])
   end
 
+  def rerun_reducers
+    duration = 3.hours
```

Comment:
Is this duration a constant we want to always enforce? Seems like it could be too long for smaller reducibles or ones in testing which would often need rerunning as they're tweaked.

Reply:
It's probably NOT a constant that we'll always want the same value for. I just couldn't think of a good way to provide UI to let the user select this.

Reply:
Maybe it'd be good to come up with a heuristic based on the number of extracts that exist?